app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,20 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
```

Comment:
There's a typo in the parameter name `hummer_level`. It should be corrected to `hammer_level`.